tests/test.ts

```diff
@@ -3,4 +3,19 @@ import { expect, test } from '@playwright/test';
 test('room code page has expected h1', async ({ page }) => {
 	await page.goto('/room/WXYZ');
 	await expect(page.getByRole('heading', { name: 'Room: WXYZ' })).toBeVisible();
-});
\ No newline at end of file
+});
+
+test('CreateRoom and JoinRoom render correctly, Join code entry validation', async ({ page }) => {
+	await page.goto('/');
+	await expect(page.getByRole('button', { name: 'Create a New Room' })).toBeVisible();
```

Comment:
One approach that I have seen in the past to try to combat UI test fragility is to add "Test IDs" to the DOM as [data attributes](https://www.w3schools.com/tags/att_data-.asp) and query for those.

The idea there is that you give the UI tests something that is designed for them to look for instead of letting them infer intention from other aspects of the DOM.

Just some food for thought